src/packages/admin-ui/src/router.tsx

```diff
@@ -59,7 +61,7 @@ export const Router = () => {
 			// We need to blend their custom routes in at the top so they can override us if they want.
 			const routes = await customPages.routes();
 			setRouter(
-				createBrowserRouter([...routes, ...defaultRoutes], {
+				createBrowserRouter([...loadAuthRoutes(), ...routes, ...defaultRoutes], {
```

Comment:
I think like the default ones we'll want to let the developer's routes take precedence:
```suggestion
				createBrowserRouter([...routes, ...loadAuthRoutes(), ...defaultRoutes], {
```